src/vs/platform/actions/browser/menuEntryActionViewItem.ts

```diff
@@ -313,4 +313,71 @@ export class LabeledMenuItemActionItem extends MenuEntryActionViewItem {
 		super.dispose();
 	}
 }
+export class LabeledNotebookMenuItemActionItem extends MenuEntryActionViewItem {
```

Comment:
Please add a doc comment explaining why this is necessary and how it differs from the LabeledMenuItemActionItem

Reply:
Added

Reply:
is this really notebook only?

Reply:
Yes -- it's called from notebook.compoent.ts

Reply:
I like this name `MaskedLabeledMenuItemActionItem` better

Reply:
I'll rename it.